bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableTable.java

```diff
@@ -154,16 +167,21 @@ public final Configuration getConfiguration() {
   /** {@inheritDoc} */
   @Override
   public HTableDescriptor getTableDescriptor() throws IOException {
-    try (Admin admin = this.bigtableConnection.getAdmin()) {
+    try (
+        NonThrowingCloseable ss =
+            TRACER.spanBuilder("BigtableTable.getTableDescriptor").startScopedSpan();
```

Comment:
Whats the difference between the spans created here and the top level spans created at the operation level?

Reply:
These names are more HBase-y than the Operation level.  Customers using HBase will need this translation.  This also gives us insight into how much time is spent on the HBase -> proto conversion.